admin/class-metabox.php

```diff
@@ -33,19 +30,18 @@ public function __construct() {
 	 * the main meta box definition array in the class WPSEO_Meta() as well!!!!
 	 */
 	public static function translate_meta_boxes() {
-		self::$meta_fields['general']['snippetpreview']['title'] = __( 'Snippet Preview', 'wordpress-seo' );
+		self::$meta_fields['general']['snippetpreview']['title'] = __( 'Snippet', 'wordpress-seo' );
```

Comment:
Because it is no longer a preview of the snippet. It is the place to edit the snippet (since the input fields for title and metadescription are now hidden).